source/guide_keycloak.rst

```diff
@@ -1,6 +1,6 @@
 .. highlight:: console
 
-.. author:: Julius Künzel
+.. author:: Christian Strauch
```

Comment:
is there any reason why you're removing the credit of the original guide's author here?

Reply:
Oh, absolutely not - apologies. Let's add him back in.